src/database/postgres_test.go

```diff
@@ -0,0 +1,253 @@
+package database
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"reflect"
+	"strings"
+	"testing"
+	"time"
+
+	pb "accretional.com/semantifly/proto/accretional.com/semantifly/proto"
+	"github.com/go-pg/pg/v10"
+	"github.com/jackc/pgx/v5"
+	_ "github.com/lib/pq"
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+func setupPostgres() error {
+	err := os.Chdir("../..")
+	if err != nil {
+		return fmt.Errorf("Failed to change directory: %v", err)
+	}
+
+	cmd := exec.Command("bash", "setup_postgres.sh")
+
+	var stdout, stderr bytes.Buffer
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+
+	err = cmd.Run()
+	if err != nil {
```

Comment:
correct me if I'm wrong but since you didn't ```set -e``` I think your script won't actually return an error here if it fails

Reply:
go docs say "The returned error is nil if the command runs, has no problems copying stdin, stdout, and stderr, and exits with a zero exit status." the script redirects stderr to stdout (which shouldn't matter from that description) and can't ever exit with nonzero status so I think Fred is right

Reply:
Yeah that's true. Added it. And some more checks in the script